packages/shared/styles/components/SfBullets.scss

```diff
@@ -0,0 +1,29 @@
+@import '../variables';
+
+$border-radius: 50%;
+$inactive-size: 8px;
+$active-size: 10px;
+$inactive-margin: 6px;
+$active-margin: 5px;
+$inactive-color: $c-gray-primary;
+$active-color: $c-green-secondary;
```

Comment:
you called inactive the default state, right?
imo you can remove this `inactive` prefix, may be just `$bullets-size` and `$bullets--active-size` for example...